util.js

```diff
@@ -28,41 +28,27 @@ debugwarn.log = console.warn.bind(console)
 /** @function */
 const isBinaryFileAsync = module.exports.isBinaryFileAsync = Promise.promisify(require('isbinaryfile'))
 
-function removePassword (input) {
+/** @function */
+const removePassword = function (input) {
   return input.replace(/(-P |pass:|\/p|-pass )([^ ]+)/, function (match, p1, p2) {
     return `${p1}***`
   })
 }
 
 /** @function */
 module.exports.execFileAsync = function (file, args, options) {
-  if (debuglog.enabled) {
-    debuglog(`Executing ${file} ${args == null ? '' : removePassword(args.join(' '))}`)
-  }
+  debuglog('Executing...', file, args && Array.isArray(args) ? removePassword(args.join(' ')) : '')
```

Comment:
@develar oh I thought if `debuglog` is not enabled then nothing should be emitted from `debuglog`... Or do we need to check `debuglog.enabled` every time we try to log something?